### PR TITLE
Add configuration to ignore certain copywrite headers

### PR DESCRIPTION
**Pull request by nywilken**

### CODE REVIEW

**Comment by tenthirtyam** [comment]
@lbajolet-hashicorp Could you review?

**Review by JenGoldstrich — APPROVED** [review]

File: .copywrite.hcl [hunk 1]
```diff
@@ -0,0 +1,9 @@
+project {
+  license = "MPL-2.0"
+  copyright_year = 2013
```
Comment (JenGoldstrich) [inline .copywrite.hcl:3]:
```suggestion
  copyright_year = 2023
```

**Review by sylviamoss — APPROVED** [review]

File: .copywrite.hcl (hunk 1 above)
Comment (nywilken) [reply .copywrite.hcl]: Ah I see where this change came from. Good catch. This year should actually match the year in the LICENSE file which is 2021. I'll update this and merge

File: .copywrite.hcl [hunk 2]
```diff
@@ -0,0 +1,9 @@
+project {
+  license = "MPL-2.0"
+  copyright_year = 2021
+  header_ignore = [
+    "*.hcl2spec.go", # generated code specs, since they'll be wiped out until we support adding headers to generated files.
+    "**/test-fixtures/**",
+    "examples/**",
```
Comment (nywilken) [inline .copywrite.hcl:7]: So I see that the supervisor examples are actually nested within the builder directories so this requires a little change.

Comment (tenthirtyam) [reply .copywrite.hcl:7]: `"**/examples/**"` ?

Comment (nywilken) [reply .copywrite.hcl]: done

**Pull request merged**